src/module/Ui/templates/entity/view/course-page.twig

```diff
@@ -22,7 +22,7 @@
 
 <div class="row">
     <div class="col-xs-12">
-        <h2 class="heading-content">{{ revision.get('title') }}</h2>
+        <h1 class="heading-content">{{ revision.get('title') }}</h1>
```

Comment:
there should only be one h1 per page which is usually the page header. is this the only h1 on that page?